warehouse/integrations/bigquery/testhelper/setup.go

```diff
@@ -34,3 +44,35 @@ func IsBQTestCredentialsAvailable() bool {
 	_, err := GetBQTestCredentials()
 	return err == nil
 }
+
+func RecordsFromWarehouse(
+	t testing.TB,
+	db *bigquery.Client,
+	query string,
+) [][]string {
+	t.Helper()
+
+	it, err := db.Query(query).Read(context.Background())
+	require.NoError(t, err)
+
+	var records [][]string
+	for {
+		var row []bigquery.Value
+		err = it.Next(&row)
+		if err == iterator.Done {
```

Comment:
I think by default it would make sense to avoid error comparisons in favour of `errors.Is` even when using a client directly. It would save us the trouble of forgetting to refactor this in case we were to use a wrapper or a different client that wraps the error. wdyt?

Reply:
Yes.